### DISCUSSION TITLE
datasets/segment/carparts-seg/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#12751] [Docs]
# datasets/segment/carparts-seg/

Explore the Carparts Segmentation using Ultralytics YOLOv8 Dataset, a large-scale benchmark for Vehicle Maintenance, and learn how to train a YOLO model using it.

https://docs.ultralytics.com/datasets/segment/carparts-seg/

**Comment by SHAHFAISAL80** [comment]
hi how i test this vehicle parts for any random dataset

**Reply by glenn-jocher** [reply]
@SHAHFAISAL80 to test vehicle parts segmentation on any random dataset, you can modify the dataset configuration file (YAML) to point to your dataset's paths and classes. Then, use the following command to train the model:

```bash
yolo segment train data=your_dataset.yaml model=yolov8n-seg.pt epochs=100 imgsz=640
```

For more details, visit the [Ultralytics documentation](https://docs.ultralytics.com/datasets/segment/carparts-seg/).
